examples/metal-nimf-2-service-profile-connection/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5.4"
+  required_providers {
+    equinix = {
+      source  = "equinix/equinix"
+      version = ">= 1.34.0"
```

Comment:
Please update this to 1.36.4. There's a breaking API response for any versions before that in UAT1 and above.

Reply:
Done